conbench/templates/benchmark-result.html

```diff
@@ -1,83 +1,119 @@
+            Think: along the plot everything is held constant except for the benchmarked code.
+            Click a data point (grey) in the plot to see a corresponding result summary.
+          </div>
+        </div>
+      {% else %}
+        <div class="alert alert-info" role="alert">
+          Cannot display history plot for this benchmark result: {{ history_plot_info.reason_why_no_plot }}
+        </div>
+      {% endif %}
     </div>
-  {% endif %}
-  <br />
+  </div>
+  <hr class="border border-danger border-1 opacity-50">
+  <h4>raw data</h4>
   <div class="row">
-    <div class="col-md-8">
+    <div class="col-md-8 mt-3">
       {{ wtf.quick_form(update_form, id="update-form", button_map={'toggle_distribution_change': update_button_color}) }}
     </div>
   </div>
   <br />
-  <div class="row">
+  <div class="row result-raw-data-container">
     <div class="col-md-6">
       <ul class="list-group">
-        <li class="list-group-item list-group-item-primary">Benchmark details</li>
+        <li class="list-group-item list-group-item-primary">Result details</li>
         <li class="list-group-item" style="overflow-y: auto;">
-          <b>Benchmark name</b>
+          <b>benchmark name</b>
           <div align="right" style="display:inline-block; float: right;">
-            <a href="{{ url_for('app.batch', batch_id=benchmark.batch_id ) }}">{{ benchmark.display_bmname }}</a>
+            <code>{{ benchmark.display_bmname }}</code>
           </div>
         </li>
         <li class="list-group-item" style="overflow-y: auto;">
-          <b>Benchmark case permutation</b>
-          <div align="right" style="display:inline-block; float: right;">{{ benchmark.display_case_perm }}</div>
-        </li>
-        <li class="list-group-item" style="overflow-y: auto;">
-          <b>Run ID</b>
+          <b>case parameter permutation</b>
           <div align="right" style="display:inline-block; float: right;">
-            <a href="{{ url_for('app.run', run_id=benchmark.run_id) }}">{{ benchmark.run_id }}</a>
+            <code>{{ benchmark.display_case_perm }}</code>
           </div>
         </li>
         <li class="list-group-item" style="overflow-y: auto;">
-          <b>Run name</b>
+          <b>submitted as part of CI run</b>
           <div align="right" style="display:inline-block; float: right;">
-            {% if run.name %}{{ run.name }}{% endif %}
-          </div>
-        </li>
-        <li class="list-group-item" style="overflow-y: auto;">
-          <b>Run reason</b>
-          <div align="right" style="display:inline-block; float: right;">
-            {% if run.reason %}{{ run.reason }}{% endif %}
+            <a href="{{ url_for('app.run', run_id=benchmark.run_id) }}">{{ benchmark.run_id }}</a> with name
+            <code>
+              {% if run.name %}{{ run.name }}{% endif %}
+            </code> and reason
+            <code>
+              {% if run.reason %}{{ run.reason }}{% endif %}
+            </code>
```

Comment:
Two thoughts:

1. This is moving further from the API result structure. Is this something we want to do? If so it's fine, but most of this page is still largely like that schema (albeit with different section names).
2. Hardware is part of what identifies the CI run; there are multiple runs with the same id/name/reason on different hardwares.

Reply:
note: good thoughts, not quickly actionable for me

about (1) we of course need to understand what a persona needs to see

not sure what's implied by (2).